### PR TITLE
Feature/new cards form

### PR DESCRIPTION
**Pull request by fel-sad**
review please :)

### CODE REVIEW

File: form.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  <body class="body--bg-starry" data-js="body">
+    <header class="--blur-strong --gradient-font">
+      <h1>Color Quiz</h1>
+    </header>
+
+    <main>
+      <form
+        class="profile-card --blur-light"
+        arialabel="Form to create new cards"
+        class="_flexcolumn"
```
Comment (robinschubertdev) [inline form.html:23]: why does this class have a _ ?

File: js/form.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
+const inputAddCard = document.querySelector('[data-js="add-card-form"]');
+const inputQuestion = document.querySelector('[data-js="input-question"]');
+const inputAnswer = document.querySelector('[data-js="input-answer"]');
+const inputTag = document.querySelector('[data-js="input-tag"]');
+const inputSubmit = document.querySelector("[data-js=input-submit]");
+const questionFeed = document.querySelector('[data-js="questions-feed"]');
+
+// const toggleButtonAnswer = document.querySelector('[data-js="reveal-button"]');
+// const cardQuestion = document.querySelector('[data-js="card-question"]');
+// const cardAnswer = document.querySelector('[data-js="card-answer"]');
```
Comment (robinschubertdev) [inline js/form.js:10]: i think you should remove outcommented Code, since you don't use it, or?

File: js/form.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+  const formAddCard = new FormData(event.target);
+  const data = Object.fromEntries(formAddCard);
+  console.log(data);
+  const dataQuestion = data.question;
+  const dataAnswer = data.answer;
+  const dataTag = data.tag;
+  console.log(dataQuestion);
+  console.log(dataAnswer);
+  console.log(dataTag);
```
Comment (robinschubertdev) [inline js/form.js:23]:
```suggestion
  
```
console.log should always be removed

File: form.html [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+        data-js="add-card-form"
+      >
+        <p --subhead>Your question</p>
+        <textarea
+          class="input--basic --button-solid"
+          data-js="input-question"
+          name="question"
+        ></textarea>
+        <p --subhead>Your answer</p>
```
Comment (alhen002) [inline form.html:32]: class attribute is missing

File: js/form.js [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+// const toggleButtonAnswer = document.querySelector('[data-js="reveal-button"]');
+// const cardQuestion = document.querySelector('[data-js="card-question"]');
+// const cardAnswer = document.querySelector('[data-js="card-answer"]');
+
+inputAddCard.addEventListener("submit", (event) => {
+  event.preventDefault();
+
+  const formAddCard = new FormData(event.target);
+  const data = Object.fromEntries(formAddCard);
+  console.log(data);
```
Comment (robinschubertdev) [inline js/form.js:17]:
```suggestion

```
same here

**Review by alhen002 — APPROVED** [review]
all good, except missing class attribute in p element.

File: js/form.js [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+  const dataQuestion = data.question;
+  const dataAnswer = data.answer;
+  const dataTag = data.tag;
+  console.log(dataQuestion);
+  console.log(dataAnswer);
+  console.log(dataTag);
+
+  const newCard = document.createElement("newCard");
+  questionFeed.append(newCard);
+  newCard.classList.add("qcard", "--blur-light");
```
Comment (robinschubertdev) [inline js/form.js:27]: maybe  its better if you dont use shortcuts for classnames, or?

**Review by robinschubertdev — CHANGES_REQUESTED** [review]
Nice Progress Felix, just left you some tiny comments

**Pull request merged**